python/nammu/view/EditSettingsView.py

```diff
@@ -231,13 +317,17 @@ def build_projects_panel(self):
 
     def build_appearance_panel(self):
         '''
-        Create the panel that'll go in the Appearance tab. This will contain
-        fields to set the font properties.
+        Create the panel that'll go in the General tab. This should contain
+        options for choosing which server to use for validation as well as
+        default working dir.
         '''
         panel = JPanel(GridBagLayout())
         constraints = GridBagConstraints()
         constraints.insets = Insets(10, 10, 10, 10)
         panel = self.build_console_font_panel(constraints, panel)
+        panel = self.build_console_font_color_panel(constraints, panel)
+        panel = self.build_console_background_color_panel(constraints, panel)
+        panel = self.build_edit_area_font_panel(constraints, panel)
```

Comment:
This has grown in an awkward way. I would have expected to see something like:

```python
panel = JPanel(GridLayout())
...
panel.add(self.building_some_panel(), constraints)
panel.add(self.building_some_other_panel(), constraints)
...
return panel
```

or you could do:

```python
panel = JPanel(GridBagLayout())
...
for panel_builder in ['build_some_panel', 'build_another_panel', ...]:
    panel.add(getattr(self, panel_builder()), constraints)
return panel
```

Reply:
I have gone for a variant of the first option in efa952a1ab, as I think it results in clearer code. By removing the return statements in each method we can remove the `panel = ` code to simply the code block.